OpenSim/Simulation/Model/BushingForce.cpp

```diff
@@ -110,23 +110,14 @@ void BushingForce::extendConnectToModel(Model& aModel)
 {
     Super::extendConnectToModel(aModel); // base class first
 
-    string errorMessage;
+    // Look up the two bodies being connected by bushing by name in the
+    // model. TODO: use Connectors
     const string& body1Name = get_body_1(); // error if unspecified
     const string& body2Name = get_body_2();
```

Comment:
@tkuchida @msdemers  That is part of the ongoing issues:  #369 4th check box. Hopefully, this PR addresses items 1-3. Then I will start ticking off the other items.